### DISCUSSION TITLE
[Question]: How to set the access path?

### DISCUSSION THREAD

**Discussion by likaia** [danny-avila/LibreChat#4847] [Q&A]
### What is your question?

After the project is deployed, how to set the access path of the ui interface. For example, I want to set: `https://xxx.com/gpt-proxy` as the access path.

### More Details

As so said above, which configuration should be used to set the root path?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
https://www.librechat.ai/docs/configuration/dotenv#application-domains

**Reply by enzberg** [reply]
I am trying to achieve something similar. I have configured a NGINX to relay traffic via https://[our-server]/chat to LibreChat.
I have also set DOMAIN_CLIENT and DOMAIN_SERVER to a non-root path as per the documentation.

The index.html is loaded properly, however subsequent frontend requests appear to access resources (e.g. assets) via the server base path (i.e. /assets/* instead of assets/* or /chat/assets/*).


Am I missing another configuration or do I need to change the frontend build?
Thanks for the help!
